.travis.yml

```diff
@@ -15,6 +15,7 @@ cache:
     - $HOME/.composer/cache
     - $HOME/.phpbrew
     - $HOME/.npm
+    - node_modules
```

Comment:
This is risky I think :P. I understand the reasoning but the way npm works (we often do `rm -rf` locally) can create issues.

Reply:
I will remove, just wanted to speed up the things, see the number of commits added 😅

Reply:
Oh interesting, Riad might be right that it can cause troubles for PRs where the lock file gets frequently updated